changelog/1.15.0.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "1.15.0"
+ådescription: "Released on 01/20/2021"
+---
+
+### Breaking Changes ❗
+
+- web: The root dashboard PWA manifest has been removed; uninstall the
+  Coder PWA. In lieu of this, Coder is moving forward with a PWA-per-application
+  approach. To install the Code Server PWA:
+  1. Log into Coder and select your environment.
+  2. Under **Applications** click **code** to launch code-server.
+  3. Follow your browser's instructions for installing the PWA.
+- web: Deprecated **Services** in favor of a declarative approach to creating
+  environments with YAML called **Environments as Code**. This feature is slated
+  to be released with Coder v2.0 (ETA: Q1 of 2021). For now, services will
+  continue to function as-is, though this feature will not receive any
+  improvements; they will be removed with Coder v2.0
+- web: The button to create an environment from an image tag has been removed
```

Comment:
This is hardly breaking. It would be nice if there was a misc section (or if we just didn't report this at all since it's very minor).

Reply:
Sure, will remove